apps/cookbook/src/app/examples/form-field-example/form-field-example-configuration-component/form-field-example-configuration.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'cookbook-form-field-example-configuration',
+  templateUrl: './form-field-example-configuration.component.html',
+  styleUrls: ['./form-field-example-configuration.component.scss'],
+})
+export class FormFieldExampleConfigurationComponent {
+  @Input() size: 'md' | 'lg';
```

Comment:
Consider using `InputSize`
```suggestion
  @Input() size: InputSize
```